isis/tests/Fixtures.cpp

```diff
@@ -475,15 +474,17 @@ namespace Isis {
     inst += stopcc;  
     
     json nk; 
+    // ikid="-74605";
     nk["INS"+ikid.toStdString()+"_FOCAL_LENGTH"] = 11994.9988;
     nk["INS"+ikid.toStdString()+"_PIXEL_PITCH"] = 0.012;
-    nk["INS"+ikid.toStdString()+"_TRANSX"] = {-89.496, -1.0e-06, 0.012};
-    nk["INS"+ikid.toStdString()+"_TRANSY"] = {-12.001, -0.012, -1.0e-06};
-    nk["INS"+ikid.toStdString()+"_ITRANSS"] = {-1000.86, -0.0087, -83.333};
-    nk["INS"+ikid.toStdString()+"_ITRANSL"] = {7457.9, 83.3333, -0.0087};
+    nk["INS-74605_TRANSX"] = {-89.496, -1.0e-06, 0.012};
+    nk["INS-74605_TRANSY"] = {-12.001, -0.012, -1.0e-06};
+    nk["INS-74605_ITRANSS"] = {-1000.86, -0.0087, -83.333};
+    nk["INS-74605_ITRANSL"] = {7457.9, 83.3333, -0.0087};
     nk["INS"+ikid.toStdString()+"_OD_K"] = {-0.0048509, 2.41312e-07, -1.62369e-13};
     nk["BODY499_RADII"] = {3396.19, 3396.19, 3376.2};
-    nk["CLOCK_ET_-74999_847428572:52459_COMPUTED"] = "8ed6ae8930f3bd41";
+    nk["CLOCK_ET_-74999_895484264:57342_COMPUTED"] = "8ed6ae8930f3bd41";
```

Comment:
This seems odd? Does this not break other tests that use the setInstrument method for the mro cube? Or is it just left over changes that should be reverted?

Reply:
I was gonna leave to Jenkins to tell me this, but the impression I have is that the original code was somewhat busted and it would essential re-spiceinit the image but skipping the table logic and hitting the Spice kernels because of this error and this test simply exposed the issue since it actually needing these variables. Does kinda break the set instrument paradigm so I might change this to it's own CTX fixture.